functionaltest/src/test/java/TestRunner.java

```diff
@@ -0,0 +1,41 @@
+import cucumber.api.CucumberOptions;
+import cucumber.api.testng.TestNGCucumberRunner;
+import cucumber.api.testng.CucumberFeatureWrapper;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+@CucumberOptions(
+    features = "src/test/resources/features",
+    glue = {"stepdefs"},
+    tags = {"@registerTest, @loginTest, @updateTest"},
+    plugin = {
+        "pretty",
+        "html:target/cucumber-reports/cucumber-pretty",
+        "json:target/cucumber-reports/CucumberTestReport.json",
```

Comment:
It looks like it's duplicated with the plugin settings in pom.xml?

Reply:
Just tested, without specifying here the reports does not get generated.

Reply:
Then the plugin in the pom.xml isn't needed?